spec/lib/bolt_spec/conn.rb

```diff
@@ -56,5 +56,35 @@ def conn_inventory
       end
       { "groups" => groups }
     end
+
+    def root_config
+      { 'modulepath' => File.join(__dir__, '../fixtures/apply'),
```

Comment:
We shouldn't hard-code this modulepath in the Conn module.

Reply:
Yeah, given that we are not going to run against puppet 5 and puppet 6 targets I don't think it makes sense to share these configs in the `Conn` class.